src/api/app/views/webui2/webui/request/_maintainer_link.html.haml

```diff
@@ -0,0 +1,6 @@
+- if maintainers.size <= 3
+  = 'Package maintainer'.pluralize(maintainers.size)
```

Comment:
[nitpick] I would put `:` after `Package maintainers`:


```
Package maintainers: user_1, user_2
```

Reply:
I agree. I will add it in!

Reply:
![Screenshot from 2019-03-08 14-17-10](https://user-images.githubusercontent.com/22715037/54057274-b4ce1e00-41af-11e9-8fa8-8f18bbb20780.png)